onnxruntime/core/providers/cpu/tensor/unsqueeze.cc

```diff
@@ -65,9 +65,8 @@ Status UnsqueezeBase::PrepareCompute(OpKernelContext* ctx, Prepare& p) const {
 Status Unsqueeze::Compute(OpKernelContext* ctx) const {
   Prepare p;
   ORT_RETURN_IF_ERROR(PrepareCompute(ctx, p));
-
+  ORT_ENFORCE(nullptr != p.input_tensor && nullptr != p.output_tensor, "Input and output must not be NULL");
```

Comment:
PrepareCompute already has an ORT_ENFORCE for the input tensor. Can add the same for output tensor as well in the same function?

Reply:
Done.